scripts/start.js

```diff
@@ -0,0 +1,7 @@
+const execute = require('child_process').execSync
```

Comment:
Should we add the `#!/usr/bin/env node` here?

Reply:
Actually, this is already a JS file and executed with `node` so I don't think this is needed. Unless, of course, you say you want this executable as a plain text file.

Reply:
Might be nice to include that as an option, in case someone ever does want to.

I'd find it weird to have to use `node scripts/start.js` rather than just `scripts/start.js` directly.

Reply:
Ah sure. I thought this was rather a thing to do when you do indeed have a plain text file (without .js extension). I'll update it :)